pkg/users/accounts-list.js

```diff
@@ -18,72 +18,277 @@
+                      isDisabled={account.uid === 0 || !account.loggedIn}
+                      onClick={() => logoutAccountDialog(account)}>
+            {_("Log user out")}
+        </DropdownItem>,
+        <DropdownSeparator key="separator-1" />,
+        <DropdownItem key="lock-account"
+                      isDisabled={account.isLocked}
+                      onClick={() => lockAccountDialog(account)}>
+            {_("Lock account")}
+        </DropdownItem>,
+        <DropdownItem key="delete-account"
+                      isDisabled={account.uid === 0}
+                      className={account.uid === 0 ? "" : "delete-account-red"}
+                      onClick={() => delete_account_dialog(account)}>
+            {_("Delete account")}
+        </DropdownItem>,
+    );
+
+    const kebab = (
+        <Dropdown toggle={<KebabToggle onToggle={isOpen => setKebabOpen(isOpen)} />}
+                isPlain
+                isOpen={isKebabOpen}
+                position="right"
+                dropdownItems={actions} />
+    );
+    return kebab;
+};
+
+const getAccountRow = (account, current, groups) => {
+    const admins = ['sudo', 'root', 'wheel'];
+    const userGroups = [];
+
+    groups.forEach((group) => {
+        if (group.userlist.find(accountName => accountName === account.name)) {
+            const isAdmin = !!admins.find(adm => adm === group.name);
+            userGroups.push({ name: group.name, members: group.userlist.length, isAdmin: isAdmin });
+        }
+    });
+
+    userGroups.sort((a, b) => {
+        if (a.isAdmin)
+            return -1;
+        if (b.isAdmin)
+            return 1;
+        if (a.members === b.members)
+            return a.name.localeCompare(b.name);
+        else
+            return b.members - a.members;
+    });
 
-        cockpit.location.go([account.name]);
+    const userGroupLabels = userGroups.map(group => {
+        const color = group.isAdmin ? "gold" : "cyan";
+        return (
+            <Label key={group.name} variant="filled" color={color}>
+                {group.name}
+            </Label>
+        );
+    });
+
+    let loginText = "";
+    let loginSortKey = null;
+    if (account.loggedIn) {
+        loginText = _("Logged in");
+        loginSortKey = "logged in";
+    } else if (!account.lastLogin) {
+        loginText = _("Never logged in");
+        loginSortKey = "never";
+    } else {
+        loginSortKey = new Date(account.lastLogin);
+        loginText = timeformat.dateTime(loginSortKey);
     }
 
-    return (
-        <Card onClick={click} onKeyPress={click} isSelectable isCompact>
-            <CardBody className="cockpit-account">
-                <UserIcon className="cockpit-account-pic" />
-                <div className="cockpit-account-real-name">{account.gecos.split(',')[0]}</div>
-                <div className="cockpit-account-user-name">
+    const columns = [
+        {
+            title: (
+                <span>
```

Comment:
Seems like the span is not needed. If you just want it for wrapper, just use empty <>

Reply:
The span is kind of useful here. This is what happened when I changed it to fragment: 

![image](https://user-images.githubusercontent.com/74668142/203023009-940f9118-10dd-48b5-a3fe-846adacb9e14.png)

(on phone the current account badge is stretched)